letsencrypt/scripts/main.py

```diff
@@ -17,6 +17,7 @@
 
 from letsencrypt.client import configuration
 from letsencrypt.client import client
+from letsencrypt.client import dns_authenticator as dns
```

Comment:
I would expect `dns` to be the `dnspython`, aliasing is slightly confusing and doesn't gain anything